common/authorization/interceptor.go

```diff
@@ -45,6 +45,13 @@ type (
 	contextKeyAuthHeader   struct{}
 )
 
+type (
+	// AudienceGetter returns JWT audience for a given request
+	AudienceGetter interface {
```

Comment:
nit: the comment implies this is only for JWT and the name feels clunky - do we expect to have more JWT specific methods? maybe call this JwtMapper (or JWTMapper - depending on what our convention is)?

Reply:
Renamed to `JWTAudienceMapper`.